plugins/modules/django_check.py

```diff
@@ -19,12 +19,13 @@
   - community.general.attributes
   - community.general.django
 options:
-  database:
+  databases:
     description:
       - Specify databases to run checks against.
       - If not specified, Django does not run database tests.
```

Comment:
I'd mention the rename here as well, so that users which have to support multiple versions of the collection know that they have to use `database` for compatibility with older versions:
```suggestion
      - If not specified, Django does not run database tests.
      - The parameter has been renamed to O(databases) in community.general 11.3.0. The old name is still available as an alias.
```